runner.py

```diff
@@ -1084,87 +1076,130 @@ def end_phase(self, phase):
     def run_flows(self):
         config = GlobalConfig().config
         # run the flows
-        for el in self._usage_scenario['flow']:
-            print(TerminalColors.HEADER, '\nRunning flow: ', el['name'], TerminalColors.ENDC)
+        ps_to_kill_tmp = []
+        ps_to_read_tmp = []
+        flow_id = 0
+        flows_len = len(self._usage_scenario['flow'])
+        while flow_id < flows_len:
+            flow = self._usage_scenario['flow'][flow_id]
+            ps_to_kill_tmp = []
```

Comment:
I would use the `.clear()` here as it was confusing as it looked like you were redefining from line 1079